packages/web-embed-api/src/index.ts

```diff
@@ -265,8 +265,14 @@ const FormsortWebEmbed = (
   ) => {
     // We overwrite `formsortOrigin` because `onWindowMessage` will read it
     formsortOrigin = formsortOrigin || `https://${clientLabel}.${DEFAULT_FLOW_DOMAIN}`;
+    let url;
+
+    if (config.origin) {
+      url = `${formsortOrigin}/client/${clientLabel}/flow/${flowLabel}`;
+    } else {
+      url = `${formsortOrigin}/flow/${flowLabel}`;
+    }
```

Comment:
Don't understand this logic, can you please explain? (ideally in comment for future clueless people like me 😅 )